.github/workflows/bake-aarch64.yml

```
@@ -92,6 +92,15 @@ jobs:
           targets: ${{ matrix.target }}
           push: ${{ inputs.no-push != true }}
           provenance: false
+      - name: Update DockerHub Description
+        if: inputs.no-push != true
+        continue-on-error: true
+        uses: peter-evans/dockerhub-description@432a30c9e07499fd01da9f8a49f0faf9e0ca5b77
+        with:
+          username: ${{ secrets.BALENAIMAGES_USER }}
+          password: ${{ secrets.BALENAIMAGES_TOKEN }}
+          repository: aarch64-alpine
```

```suggestion
          repository: balenalib/aarch64-alpine
```

Full name would be needed

---

Good catch, that actually explains the error!

---

Unfortunately we still have the same error
https://github.com/balena-io-library/base-images/actions/runs/14892091622

Will check if the token has been used or not